docs/architecture/adr-043-basenft-module.md

```diff
@@ -0,0 +1,126 @@
+# ADR 43: BaseNFT Module
+
+## Changelog
+
+- 05.05.2021: Initial Draft
+
+## Status
+
+DRAFT
+
+## Abstract
```

Comment:
Thanks @HuLaTown! But this ADR does not mention or talk about what NFTs are, why they're needed in the SDK, what owners are, what IDs are and their relationships. Any chance you can expand upon this ADR a bit more please?

Reply:
Got it! Thank you Alex, Fede and Marko! Will expand more on the introduction part.